Top K Frequently Repeating.py

```diff
@@ -0,0 +1,52 @@
+# TC: O(NlogK)
+# SC : O(N)
+class Solution:
+    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
+        hashmap = {}
+        hq = []
+        for num in nums:
+            if num not in hashmap:
+                hashmap[num] = 1
+                
+            else:
+                hashmap[num]+=1               
+        
+        
+        for num,count in hashmap.items():
+            heappush(hq,(count,num))
+            if len(hq) > k:
+                heappop(hq)
+                
+        result = []
+        
+        for i in range(len(hq)):
+            result.append(hq[i][1])
+            
+        return result
+    
+# TC : O(N)
+# SC: O(N)
+class Solution(object):
```

Comment:
Answer all given questions according to the template
Below 2 answers are missing
Did it successfully run on leetcode:
Any issues while solving this problem: